spec/yarn.lock

```diff
@@ -229,7 +229,7 @@ dashdash@^1.12.0:
     safe-buffer "^5.1.1"
     xml2js "^0.4.17"
   optionalDependencies:
-    abstract-socket "github:saghul/node-abstractsocket#35b1b1491fabc04899bde5be3428abf5cf9cd528"
+    abstract-socket "^2.0.0"
```

Comment:
I'm not sure what's going on with that the old version of this line, that old version seems strange to me. But unless this change is on-topic to this PR, it should be handled separately